zebra-chain/src/transaction/lock_time.rs

```diff
@@ -1,66 +1,86 @@
+//! Transaction LockTime.
+
 use std::{convert::TryInto, io};
 
 use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
 use chrono::{DateTime, TimeZone, Utc};
 
-use crate::block;
+use crate::block::{self, Height};
 use crate::serialization::{SerializationError, ZcashDeserialize, ZcashSerialize};
 
 /// A Bitcoin-style `locktime`, representing either a block height or an epoch
 /// time.
 ///
 /// # Invariants
 ///
-/// Users should not construct a `LockTime` with:
-///   - a `block::Height` greater than MAX_BLOCK_HEIGHT,
+/// Users should not construct a [`LockTime`] with:
+///   - a [`block::Height`] greater than [`LockTime::MAX_HEIGHT`],
 ///   - a timestamp before 6 November 1985
-///     (Unix timestamp less than MIN_LOCK_TIMESTAMP), or
+///     (Unix timestamp less than [`LockTime::MIN_TIMESTAMP`]), or
 ///   - a timestamp after 5 February 2106
-///     (Unix timestamp greater than MAX_LOCK_TIMESTAMP).
+///     (Unix timestamp greater than [`LockTime::MAX_TIMESTAMP`]).
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
 pub enum LockTime {
-    /// Unlock at a particular block height.
+    /// A block height after which the transaction can be included in the block chain.
```

Comment:
"After" is unclear, it could mean "greater than" or "greater than or equal to" (a transaction in the block "after" this block header, or a transaction in the next block):
```suggestion
    /// This transaction can be included in blocks with heights greater that this lock height.
```